docker-compose.yml

```diff
@@ -6,38 +6,56 @@ services:
         command: >
             /bin/bash -c "
                 until echo > /dev/tcp/postgres/5432; do sleep 1; done
-                npx pubsweet server
```

Comment:
cross-check if the `npx` at https://github.com/elifesciences/elife-xpub-formula/blob/f2f9bb2a0fa85cb210832ae93a8b5d88f470a75b/salt/elife-xpub/scripts/create-database.sh#L5 should (separately) be changed too?

Reply:
We should probably remove this script when we configure RDS, so I think that script will become redundant. @tamlyn  ?

Reply:
Running `npx pubsweet setupdb` is fine. It would only need to be run the first time a database is created.

Reply:
Will ignore it for now